tests/tests/tier0/bluechi-is-online-agent/test_bluechi_is_online_agent.py

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright Contributors to the Eclipse BlueChi project
+#
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+from typing import Dict
+
+from bluechi_test.config import BluechiAgentConfig, BluechiControllerConfig
+from bluechi_test.machine import BluechiAgentMachine, BluechiControllerMachine
+from bluechi_test.test import BluechiTest
+
+NODE_FOO = "node-foo"
+
+
+def check_execs(node: BluechiAgentMachine):
+    agent_output = node.bluechi_is_online.agent_is_online()
+    assert agent_output
```

Comment:
Currently the test checks only initial state, I'd extend the test to perform:

1. Run `bluechi-is-online agent` and verify it returns 0 (agent should be started and connected)
2. Stop bluechi-agent on the `node-foo`
3. Run `bluechi-is-online agent` again and verify it returns non-zero  (as we previously stopped the agent)

Reply:
Done